### ISSUE TITLE
tab_index should match the tabindex attribute

### ISSUE THREAD

**Issue by jkotests** [watir/watir#207]
It appears that the tabindex attribute is being treated as tab_index in watir-webdriver. This seems to cause two issues:
1) Element#tab_index will return nil when an element has the tabindex attribute
2) :tab_index cannot be used to successfully locate an element that has a tabindex attribute

For example, given the html:

``` html
<input type="checkbox" value="1" tabindex="8">
```

I would expect the tab_index method to return "8". However, you get:

``` ruby
browser.checkbox.tab_index
#=> nil

browser.checkbox.tabindex
#=> NoMethodError: undefined method `tabindex' for #<Watir::CheckBox:0x34e5598>
```

When the :tab_index is used in a locator, the xpath generated is looking for the tab-index attribute (seen when $DEBUG=true):

``` ruby
b.checkbox(:tab_index => '8').exists?
#=> ...
#=> :xpath=>".//input[@tab-index='8' and translate(@type,'ABCDEFGHIJKLMNOPQRSTUVWXYZ','abcdefghijklmnopqrstuvwxyz')='checkbox']"
#=> ...
```

To my knowledge, html only supports the 'tabindex' attribute rather than the 'tab_index' attribute (http://www.w3schools.com/tags/att_global_tabindex.asp).

Therefore, I had expected watir-webdriver to either:
A) Use tabindex (as a Element method and valid locator) or
B) Translate tab_index to tabindex

**Comment by titusfortner** [comment]
The fix for #28 did not address this issue.

I think the solution is to create an `html_element.rb` file with alias methods similar to what we are doing for `colspan` and `rowspan` [here](https://github.com/watir/watir/blob/master/lib/watir/elements/table_cell.rb)

**Comment by titusfortner** [comment]
Closing in favor of #600 since that is the general use case of this one.
